working with newer magento

adeptofvoltron opened the pull request:
after that merge...reverte in main project commit hash: 8eaca89c3736fdaca2dc7b91867fbcb9c238a643

and run composer update

adeptofvoltron committed:
merge newer version

laurent35240 reviewed (changes requested):
I did a test locally and I did not receive the emails (in my test for confirming an order)

Here are the logs I see for emarsys
![Admin Magento   ILLICADO](https://user-images.githubusercontent.com/473193/59597730-d61d9f00-90fa-11e9-9950-b0ff3e5a3496.png)

adeptofvoltron committed:
update sendEmail

I think it would be better to find proper values for that entries...but probably we would have to ask emarsys and now there is no time for it
